### ISSUE TITLE
[Question] Text for Input Element Binding

### ISSUE THREAD

**Issue by eibanez** [vega/vega-lite#3350]
It looks like the text next to the generated input elements when using "bind" are the autogenerated names. Is there a way to override this?

I also see that there is a `<label>` created after the sliders in the HTML. Is there a way to inject text there?

Examples:
https://vega.github.io/vega-lite/docs/bind.html#input-element-binding

**Comment by jwoLondon** [comment]
Try adding

     "name":"myLabel"

to the input `bind` object. That should do the job.

**Comment by eibanez** [comment]
Oh, I just caught the "name" option [here](https://vega.github.io/vega-lite/docs/selection.html#type). Wasn't clear to me. Thanks!

**Comment by eibanez** [comment]
This works, but throws a validation warning:

```
{
  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
  "data": {
    "url": "data/cars.json",
    "format": {
      "type": "json"
    }
  },
  "selection": {
    "org": {
      "type": "single",
      "fields": [
        "Origin"
      ],
      "bind": {
        "input": "select",
        "name": "test",
        "options": [
          "Europe",
          "Japan",
          "USA"
        ]
      },
      "on": "click",
      "resolve": "global",
      "empty": "all"
    }
  },
  "mark": "point",
  "encoding": {
    "x": {
      "field": "Horsepower",
      "type": "quantitative"
    },
    "y": {
      "field": "Miles_per_Gallon",
      "type": "quantitative"
    },
    "color": {
      "condition": {
        "selection": "org",
        "field": "Cylinders",
        "type": "ordinal"
      },
      "value": "grey"
    }
  }
}
```

**Comment by ijlyttle** [comment]
Hi,

I see that this has been addressed in vega/vega-lite#3352, with vega/vega-typings#14. However, when I paste the  👆sample spec into the vega-lite editor (using vega-lite 2.6.0), I still get the validation warning.

Thanks!
